### ISSUE TITLE
Improve Error Handling when CyberExchange is down

### ISSUE THREAD

**Issue by zacharyshupp** [zacharyshupp/SAM.DoDCyberExchange#1]
**Describe the bug**
When invoking any functions the Get-CEItem returns the below error.

```PowerShell
Cannot validate argument on parameter 'Uri'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - PowerShell Version: 5.1 and 7.1

**Comment by zacharyshupp** [comment]
Found the certificate for public.cyber.mil is expired. Adding error handling to support.

**Comment by zacharyshupp** [comment]
This issue is resolved and will be deployed in v0.2.1.
